fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResource.java

```diff
@@ -178,30 +180,30 @@
-            final DefaultToApiJsonSerializer<LoanScheduleData> loanScheduleToApiJsonSerializer,
-            final ApiRequestParameterHelper apiRequestParameterHelper, final FromJsonHelper fromJsonHelper,
-            final PortfolioCommandSourceWritePlatformService commandsSourceWritePlatformService,
-            final CalendarReadPlatformService calendarReadPlatformService, final NoteReadPlatformServiceImpl noteReadPlatformService,
-            final PortfolioAccountReadPlatformService portfolioAccountReadPlatformServiceImpl,
-            final AccountAssociationsReadPlatformService accountAssociationsReadPlatformService,
-            final LoanScheduleHistoryReadPlatformService loanScheduleHistoryReadPlatformService,
-            final AccountDetailsReadPlatformService accountDetailsReadPlatformService,
-            final EntityDatatableChecksReadService entityDatatableChecksReadService,
-            final BulkImportWorkbookService bulkImportWorkbookService,
-            final BulkImportWorkbookPopulatorService bulkImportWorkbookPopulatorService) {
+        final LoanProductReadPlatformService loanProductReadPlatformService,
+        final LoanDropdownReadPlatformService dropdownReadPlatformService, final FundReadPlatformService fundReadPlatformService,
+        final ChargeReadPlatformService chargeReadPlatformService, final LoanChargeReadPlatformService loanChargeReadPlatformService,
+        final CollateralReadPlatformService loanCollateralReadPlatformService,
+        final LoanScheduleCalculationPlatformService calculationPlatformService,
+        final GuarantorReadPlatformService guarantorReadPlatformService,
+        final CodeValueReadPlatformService codeValueReadPlatformService, final GroupReadPlatformService groupReadPlatformService,
+        final DefaultToApiJsonSerializer<LoanAccountData> toApiJsonSerializer,
+        final DefaultToApiJsonSerializer<LoanApprovalData> loanApprovalDataToApiJsonSerializer,
+        final DefaultToApiJsonSerializer<LoanScheduleData> loanScheduleToApiJsonSerializer,
+        final ApiRequestParameterHelper apiRequestParameterHelper, final FromJsonHelper fromJsonHelper,
+        final PortfolioCommandSourceWritePlatformService commandsSourceWritePlatformService,
+        final CalendarReadPlatformService calendarReadPlatformService, final NoteReadPlatformServiceImpl noteReadPlatformService,
+        final PortfolioAccountReadPlatformService portfolioAccountReadPlatformServiceImpl,
+        final AccountAssociationsReadPlatformService accountAssociationsReadPlatformService,
+        final LoanScheduleHistoryReadPlatformService loanScheduleHistoryReadPlatformService,
+        final AccountDetailsReadPlatformService accountDetailsReadPlatformService,
+        final EntityDatatableChecksReadService entityDatatableChecksReadService, RateReadService rateReadService,
+        final BulkImportWorkbookService bulkImportWorkbookService,
```

Comment:
Kindly make RateReadService rateReadService as the last parameter of the method

Reply:
Ok.